### ISSUE TITLE
overfitting

### ISSUE THREAD

**Issue by vivi0604** [maudzung/SFA3D#36]
I'm really appreciate you for the nice source code.  With my own datasets, i got the nice loss of training , but in validation loss increase , i think may be the model is overfitting . Can you recommand me some way to improve the accuracy ?
lr=0.0006, epochs=120 ,6630 frames for training
![image](https://user-images.githubusercontent.com/38038575/111722994-dafa1e80-889d-11eb-82de-4fd1c7512485.png)
